src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -124,12 +124,12 @@ public StateFlow build(FlowState.Main initialState) {
             .transitionTo(CLAIM_SUBMITTED).onlyIf(claimSubmittedNoRespondentRepresented
                                                       .or(claimSubmittedOnlyOneRespondentRepresented))
             .state(CLAIM_SUBMITTED)
-            .transitionTo(CLAIM_ISSUED_PAYMENT_SUCCESSFUL).onlyIf(paymentSuccessful)
-            .transitionTo(CLAIM_ISSUED_PAYMENT_FAILED).onlyIf(paymentFailed)
+                .transitionTo(CLAIM_ISSUED_PAYMENT_SUCCESSFUL).onlyIf(paymentSuccessful)
```

Comment:
If changes in StateFlowEngine are only for formatting, fyi there is another PR being forwarded by Harry which especially does it.. so to avoid conflicts, please revert the indentation/styling changes in the file and keep only the code changes.